cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionFilter.java

```diff
@@ -141,6 +141,32 @@ public PartitionFilter build() {
       Preconditions.checkState(!map.isEmpty(), "Partition filter cannot be empty.");
       return new PartitionFilter(map);
     }
+
+    /**
+     * @return <tt>true</tt> if the underlying map contains no key-value mappings
```

Comment:
Javadoc should not explain by means of the implementation. It should always explain by means of what the user sees. In this case, it would be better to say" returns true if this filter is always true. Or better: if this filter imposes no conditions